querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java

```diff
@@ -689,4 +690,14 @@ private static String emptyIfSetToBlank(String value) {
         boolean setToBlank = value == null || value.equalsIgnoreCase("BLANK");
         return setToBlank ? "" : value;
     }
+
+    private static String processBlankValues(String value) {
+        if (value == null) {
+            return null;
+        }
+        return BLANK_VALUE_PATTERN.matcher(value).replaceAll(BLANK_VALUE_REPLACEMENT);
+    }
+
+    private static final Pattern BLANK_VALUE_PATTERN = Pattern.compile("(^|,)BLANK(,|$)", Pattern.CASE_INSENSITIVE);
+    private static final String BLANK_VALUE_REPLACEMENT = "$1$2";
```

Comment:
Why have you decided to do it differently than `emptyIfSetToBlank`?
If this is a better way, why not move it all over?

Reply:
@Shredder121, the two scenarios require different behaviors, and I didn't want to break backward compatibility for the existing API:

`emptyIfSetToBlank` (existing):
- Converts `null` to an empty string
- Converts `"BLANK"` to an empty string
- Works with a single value

`processBlankValues` (new):
- Keeps `null` values as `null`
- Converts `"BLANK"` to an empty string
- Works with multiple comma-separated values (not a compatibility concern)

This change in behavior is required since the two values are treated differently by the `DatabaseMetadata` class.

If you are ok with changing the null-handling behavior of `namePrefix`, then the two functions can be merged.